src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -383,981 +211,204 @@
+	"mmorpg.chat.cant_use_multiple_same_supports": "Нельзя использовать несколько одинаковых поддерживающих самоцветов.",
+	"mmorpg.chat.cast_failed": "[Не удалось произвести заклинание]: ",
+	"mmorpg.chat.character_load_info": "НАЖМИТЕ, чтобы загрузить этого персонажа\n\nСохраненные характеристики персонажа:\n- Ваш уровень и опыт\n- Ваши заклинания, статистика, таланты, настройка панели действий\n\nНе сохраняются ваша экипировка, самоцветы, драгоценные камни и т. д.\n\nКаждый персонаж начинает с 1 уровня",
+	"mmorpg.chat.command_block_unavaliable": "Командные блоки отключены, это препятствует игре в Подземелья Mine and Slash!",
+	"mmorpg.chat.create_new_character": "Создать нового персонажа\n\nИмя должно быть уникальным",
+	"mmorpg.chat.enemy_too_close": "Вы не можете телепортироваться, когда рядом враги.",
+	"mmorpg.chat.gear_drop": "Вы не соответствуете требованиям этого предмета.",
+	"mmorpg.chat.gem_socketed": "Самоцвет вставлен",
+	"mmorpg.chat.how_to_enable_command_block": "Чтобы включить, перейдите в файл server.properties и установите enable-command-block на true.",
+	"mmorpg.chat.lack_aura_capacity": "У вас недостаточно емкости ауры, чтобы экипировать все эти ауры.",
+	"mmorpg.chat.level_up_message_down": "%1$s Уровень: %2$s > %3$s!",
+	"mmorpg.chat.level_up_message_up": "Повышение уровня!",
+	"mmorpg.chat.max_map_rarity": "Эта карта имеет максимальную возможную редкость.",
+	"mmorpg.chat.max_map_rarity_for_lvl": "Эта карта имеет максимальную возможную редкость для своего уровня.",
+	"mmorpg.chat.no_charges": "У вас нет зарядов для заклинаний.",
+	"mmorpg.chat.no_duplicate_aura": "Вы не можете экипировать одинаковые самоцветы ауры.",
+	"mmorpg.chat.no_mana": "Недостаточно ресурсов для произнесения.",
+	"mmorpg.chat.none_mage": "Оружие для Не-Магов может быть использовано.",
+	"mmorpg.chat.not_in_this_dimension": "Вы не можете произнести это заклинание в этом измерении.",
+	"mmorpg.chat.not_map": "Это не карта.",
+	"mmorpg.chat.not_mns_weapon": "Вам нужно оружие Mine and Slash (Обладающее статистикой), чтобы произнести заклинания.",
+	"mmorpg.chat.open_loot_chest": "Щелкните правой кнопкой мыши, чтобы открыть ларец с добычей!",
+	"mmorpg.chat.prof_output_slot_not_empty": "Нужен хотя бы 1 пустой слот вывода.",
+	"mmorpg.chat.prof_recipe_level_not_enough": "Уровень недостаточно высок для крафта.",
+	"mmorpg.chat.prof_recipe_not_found": "Рецепт не найден",
+	"mmorpg.chat.require_mage": "Требуется магическое оружие для использования.",
+	"mmorpg.chat.require_melee": "Требуется ближнее оружие для использования.",
+	"mmorpg.chat.require_ranged": "Требуется дистанционное оружие для использования.",
+	"mmorpg.chat.soulless_gear_info": "Это снаряжение обездушено\nДуши снаряжения содержат статистику Mine and Slash\nБездушные предметы в основном бесполезны\nДуши создаются на станциях или выпадают из мобов",
```

Предлагаю везде использовать термин "характерстики" вместо "статистики" (Stat), когда мы работаем с этим контекстом
```suggestion
	"mmorpg.chat.soulless_gear_info": "Это снаряжение обездушено\nДуши снаряжения содержат характеристики Mine and Slash\nБездушные предметы в основном бесполезны\nДуши создаются на станциях или выпадают из мобов",
```

---

исправлено